resources/js/langman.js

```diff
@@ -80,9 +80,9 @@ new Vue({
          * Add a new translation key
          */
         addNewKey(key) {
```

Comment:
I didn't create new action for adding new language, because i just add action what was created for this by you. I just display in index.blade.php button for create new language, in this metod i commented if statement, because if you scaned files and nothing new came out this alert worked as many times as there were keys and it is annoying. Here you displaying keys into loop so this is working like that. I testing this changes and everything working perfectly.